### ISSUE TITLE
Preferred extension for schemas is .graphql

### ISSUE THREAD

**Issue by benmccann** [kobylynskyi/graphql-java-codegen#97]
It looks to me like `.graphql` is the most commonly used extension for schemas: https://github.com/graphql/graphql-spec/issues/203#issuecomment-242140024

Could the project support either just `.graphql` or both `.graphql` and `.graphqls` by default?

**Comment by kobylynskyi** [comment]
@benmccann 
the codegen library works with both `.graphql` and `.graphqls`. 
You can just specify it in the maven config:
```xml
<graphqlSchemaPaths>
    <graphqlSchemaPath>${project.basedir}/src/main/resources/schema.graphql</graphqlSchemaPath>
</graphqlSchemaPaths>
```
or:
```xml
<graphqlSchemas>
    <includePattern>.*\.graphql</includePattern>
</graphqlSchemas>
```

**Comment by benmccann** [comment]
Yeah, I was just suggesting a different default to better align with the ecosystem

**Comment by kobylynskyi** [comment]
Oh, ok. You are right.
I will change the default value of `graphqlSchemas.includePattern` to support both `.graphql` and `.graphqls`

**Comment by joffrey-bion** [comment]
It seems from what I have seen that people use .graphql for query files,
and .graphqls for schema files. That's also what my IDE picked up when
using these files.

But maybe I didn't push the research enough. I'd be happy to change the
default if it reflects better the convention, but I'm not sure it's
actually the case right now.

**Comment by kobylynskyi** [comment]
The change will be released in v1.7.0 by the end of this week.

**Comment by benmccann** [comment]
Thanks!
